packages/camera/camera_windows/windows/capture_controller.cpp

```diff
@@ -39,7 +39,7 @@ CaptureControllerImpl::CaptureControllerImpl(
     : capture_controller_listener_(listener),
       media_settings_(
           PlatformMediaSettings(PlatformResolutionPreset::kMax, true)),
-      CaptureController(){};
+      CaptureController() {};
```

Comment:
All of the rest of these are minor differences in how clang-format handles this code, but the easy solution to these is to fix them all to not have the stray `;` that shouldn't be there and is causing it to have non-trivial format analysis. Every one of these should have stable formatting across versions once the incorrect `;` is gone.

Reply:
I removed the `;` and the formatter still adds the space. Is that the expected behavior?

Reply:
Yes, the space is correct.